src/renderer/app/services/environments.service.ts

```diff
@@ -474,11 +474,26 @@ export class EnvironmentsService {
       if (log.response) {
         const headers: Header[] = [];
         log.response.headers.forEach((header) => {
+          if (
+            header.key === 'content-encoding' ||
```

Comment:
I think we may need transfer-encoding added to this list to facilitate the mock this endpoint.

Reply:
As of now, transfer-encoding is not checked also while decoding messages. Maybe need to check at both decoding and header update. I'll check what is possible.
Thanks.